viewer/package.json

```diff
@@ -40,7 +41,7 @@
   },
   "scripts": {
     "start": "REACT_APP_API_BASE_URL='http://localhost:8000' react-scripts start",
-    "test": "react-scripts test",
+    "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!export-to-csv)/\"",
```

Comment:
This can go in the "jest" block.  See my dag PR for an example.

Reply:
Cool, will move it there 👍